string c_class
geometry_msgs/Point r_eye
geometry_msgs/Point l_eye
geometry_msgs/Point r_ear
geometry_msgs/Point l_ear
geometry_msgs/Point nose
geometry_msgs/Point neck
geometry_msgs/Point r_sho
geometry_msgs/Point l_sho
geometry_msgs/Point r_elb
geometry_msgs/Point l_elb
geometry_msgs/Point r_wri
geometry_msgs/Point l_wri
geometry_msgs/Point r_hip
geometry_msgs/Point l_hip
geometry_msgs/Point r_knee
geometry_msgs/Point l_knee
geometry_msgs/Point r_ank
geometry_msgs/Point l_ank

